Gemfile

```diff
@@ -6,3 +6,7 @@ gem 'rake'
 gem 'rspec'
 gem 'govuk-dummy_content_store', '0.0.5'
 gem 'foreman', '0.78.0'
+
+group :test do
+  gem 'pry-byebug'
```

Comment:
In other GOV.UK repos I've found both examples of this being done within [a dedicated PR](https://github.com/alphagov/publishing-api/pull/114), and examples of it being done within a [PR that was doing something else](https://github.com/alphagov/frontend/pull/1004/commits/854751da9542dc47b24eb9969cadc6d966c71a08). 
PRs create a fair bit of noise (github notifications) and overhead (additional item to deploy), which I feel are not worth having for just adding a debugging gem. IMO avoiding this is more important than having a commit not directly related to the PR here. 
That being said, it's a matter of opinion and dedicated PRs have happened before, so if one of you has strong feelings about it, I can remove it to not block this PR.